BetterBags_Vanilla.toc

```diff
@@ -9,22 +9,24 @@
 ## X-License: MIT
 ## X-Curse-Project-ID: 942432
 ## X-Wago-ID: aNDmy96o
-## OptionalDeps: LibStub, Masque, CallbackHandler-1.0, Ace3, LibSharedMedia-3.0, _DebugLog, ConsolePort, Pawn, SortBags, WagoAnalytics, ElvUI
+## OptionalDeps: LibStub, Masque, CallbackHandler-1.0, Ace3, LibSharedMedia-3.0, _DebugLog, ConsolePort, Pawn, SortBags, WagoAnalytics, ElvUI, LibWindow-1.1, LibDataBroker-1.1, LibUIDropDownMenu, AceGUI-3.0-SharedMediaWidgets
```

Comment:
See above comment on this.

Reply:
I can do as you ask. However, none of those libs are external to BetterBags. You include all of them in the pkgmeta, bundled into the libs folder, because you use all the libs in your code. Therefore, all of them must be listed in the ToC files, and they must be loaded before the core addon files –– which is what the OptDeps line does.

To prove the point, disable every addon except BetterBags, including debugger addons. Do not install external libraries as standalone versions. If any libraries are packaged with a zip, include those. Then remove the entries from the OptDeps line, or remove the whole line from the ToC. WoW should complain that dependencies are not loaded.

The difference between OptDeps and Deps is that libs or other addons listed in OptDeps **can** be loaded by *other addons*, whereas mentioning them in a Deps line means they **must** be loaded by *your addon*.

So, yes, I can make the requested changes, but doing so is incorrect. What I presented is the correct methodology for loading external libraries.

As for reverting the BWP line to a specific version, sure, that is much easier.

Reply:
I pushed the changes, including the BWP version.